Makefile

```diff
@@ -197,11 +197,12 @@ src :=  corev_apu/tb/axi_adapter.sv
         corev_apu/tb/common/SimDTM.sv                                                \
         corev_apu/tb/common/SimJTAG.sv
 
+flists := ${CVA6_REPO_DIR}/core/Flist.cva6
 # SV32 MMU for CV32, SV39 MMU for CV64
 ifeq ($(findstring 32, $(target)),32)
-    src += $(wildcard core/mmu_sv32/*.sv)
```

Comment:
A recent PR has factorize the different flist into a single one. I do not understand why questa cannot manage it like VCS ?

Reply:
I assume you mean #1012? That PR reintroduces the problem from #713 and is exactly what I am trying to solve here. The underlying problem is that Questasim elaborates all source files, even if the modules are not instantiated. When elaborating `mmu_sv32` for the CV64A6 configuration, it encounters a `Negative replication multiplier` error. The proposed solution is to prevent the elaboration of an incompatible MMU. More details in #713

Reply:
Is it too ugly to suppress the error? This will fix the issue, but might lead to actual errors being hidden. I am unaware about the details, but maybe this can be done on a per-file basis?

Reply:
Hello, thanks @MarioOpenHWGroup to propose multi solutions. The suppress 8607 seems good to me and I would suggest to open an github issue to log the workaround.